wallet-sdk/src/main/kotlin/org/stellar/walletsdk/anchor/Sep24InteractiveFlow.kt

```diff
@@ -0,0 +1,165 @@
+package org.stellar.walletsdk.util
```

Comment:
Should we move away from using SEPs in file/method names? Interactive flow == SEP 24

Reply:
Yes, I agree. I just forgot to rename the file.